sleap_roots/angle.py

```diff
@@ -0,0 +1,28 @@
+"""Get angle of each root."""
+
+import numpy as np
+
+
+def get_root_base_angle(pts: np.ndarray, base_ind=0, tip_ind=1) -> np.ndarray:
```

Comment:
Could you add type hints to the kwargs here? like:
```
base_ind: int = 0, tip_ind: int = 1
```

Also, shouldn't `tip_ind` maybe default to `-1`?